cpp/src/parquet/metadata_test.cc

```diff
@@ -180,6 +185,12 @@ TEST(Metadata, TestBuildAccess) {
 
   // Test AppendRowGroups
   auto f_accessor_2 = GenerateTableMetaData(schema, props, nrows, stats_int, stats_float);
+  
+  ASSERT_EQ(512, f_accessor_2->ToString().length());
+  ASSERT_EQ(21, static_cast<int32_t>(f_accessor_2->ToString()[0]));
```

Comment:
these tests seem a little bit opaque as to why thes values are expected.  If the thrift layer changes the serialization format it seems like they might break for no reason.  Would round tripping the data through the Make API be a better test for this?

Reply:
done.